babel/src/test/resources/sql/spark.iq

```diff
@@ -240,4 +240,47 @@ EXPR$0
 true
 !ok
 
+#####################################################################
+# REVERSE
+#
+# REVERSE(str | array)
```

Comment:
This SQL is valid in Spark:
```
spark-sql> SELECT reverse(5);
5
Time taken: 0.071 seconds, Fetched 1 row(s)
```
5 is a number.

Reply:
This support is due to spark type implicit conversion. In addition, Calcite also has a corresponding ability, although the abilities of the two are not exactly the same.

This test case has been added in `spark.iq`.